pricing-page-exercise/saffet/styles.css

```diff
@@ -0,0 +1,257 @@
+body {
+    font-family: sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+
+/* Navigation bar*/
+.nav-bar{
+    box-sizing: border-box;
+    height: 70px;
+    border: 1px transparent grey;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-left: 25px;
+    padding-right: 35px;
+    box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+}
+
+.nav-bar > p{
+    font-size: 19px; /*company name*/
+}
+
+.nav-links{
+    display: flex;
+    align-items: center;
+}
+
+.nav-links a{
+    text-decoration: none;
+}
+
+.nav-links>a{
+    padding-left: 25px;
+    color: black;
```

Comment:
Try to use HEX colors everywhere instead and RGBA when you need to have a color with opacity. This is what's used the most in real life.

Reply:
I generated from web so some lines has this situation but I will be more careful next time